src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeMapperTest.java

```diff
@@ -17,46 +17,69 @@
 import uk.gov.hmcts.reform.bulkscanprocessor.validation.model.OcrValidationWarnings;
 
 import java.time.Instant;
-import java.util.Arrays;
 import java.util.List;
 import java.util.Optional;
 
+import static java.util.Arrays.asList;
 import static java.util.stream.Collectors.toList;
 import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
 import static uk.gov.hmcts.reform.bulkscanprocessor.helper.EnvelopeCreator.getEnvelopeFromMetafile;
 import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentSubtype.COVERSHEET;
 import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentSubtype.SSCS1;
 import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentSubtype.WILL;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentType.CHERISHED;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentType.FORM;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentType.OTHER;
 
 public class EnvelopeMapperTest {
+    private static final String METAFILE_BASE_PATH = "/metafiles/valid/";
 
     @Test
-    public void should_map_zip_envelope_correctly() throws Exception {
-        InputEnvelope zipEnvelope = getEnvelopeFromMetafile();
+    public void should_map_envelope_fields_payments_andnon_scannable_items_correctly() throws Exception {
+        InputEnvelope zipEnvelope = getEnvelopeFromMetafile(METAFILE_BASE_PATH + "no-scannable-items.json");
+        String container = "container1";
+
+        Envelope dbEnvelope = EnvelopeMapper.toDbEnvelope(zipEnvelope, container, Optional.empty());
+
+        assertSameFields(dbEnvelope, zipEnvelope);
+
+        assertSamePayments(dbEnvelope, zipEnvelope);
+        assertThat(dbEnvelope.getScannableItems()).isEmpty();
+        assertSameNonScannableItems(dbEnvelope, zipEnvelope);
```

Comment:
We could also keep scannable item checking here (that check doesn't take ocr and document types into account):
```
assertSameScannableItems(dbEnvelope, zipEnvelope);
```

And then the name of the method could stay as it was.

Reply:
Same with ocr data.